src/tests/test_magic_cte.py

```diff
@@ -182,3 +182,124 @@ def test_query_syntax_error(ip):
 
     assert excinfo.value.error_type == "RuntimeError"
     assert CTE_MSG.strip() in str(excinfo.value)
+
+
+def test_comment_in_query_added_after_bracket(ip):
```

Comment:
can you add more clearer test name? I believe what we're testing is that the stored snippets do not contain comments

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/894/commits/104713b221173966cf14c3a25e66a42390dd5724)